the-telephone-app-fe/src/components/StartGame/StartGame.js

```diff
@@ -125,7 +130,29 @@ const StartGame = () => {
       });
     }
   };
+  const imgs = [htp1, htp2, htp3, htp4, htp5];
+  const titles = [
+    "JOIN GAME",
+    "TIME TO WRITE",
+    "TIME TO DRAW",
+    "WHAT IS IT?",
+    "SEE WHAT HAPPENED",
+  ];
+  const contents = [
+    "Invite your friends",
+    "Each player must write a quirky sentence",
+    "You are gonna receive a bizzare sentence to draw",
+    "Try to describe one of the crazy drawings",
+    "Watch the hilarious results of the telephone game",
+  ];
+  const [currentGuide, setCurrentGuide] = useState(0);
 
+  const handleHTP = (index) => {
+    setCurrentGuide(index);
+    setActiveButton(index);
+  };
+
+  const [activeButton, setActiveButton] = useState(null);
```

Comment:
currentGuide and activeButton is kind of duplicated, use currentGuide for button is fine.